genie/database_to_staging.py

```diff
@@ -1746,16 +1746,16 @@ def stagingToCbio(
         wes_panelids,
     )
 
-    store_fusion_files(
-        syn,
-        consortiumReleaseSynId,
-        genieVersion,
-        fusionSynId,
-        keepForCenterConsortiumSamples,
-        keepForMergedConsortiumSamples,
-        current_release_staging,
-        CENTER_MAPPING_DF,
-    )
+    # store_fusion_files(
```

Comment:
Can you delete this particular function and these commented code chunks to see how the code runs?